src/avt/VisWindow/Colleagues/avtAnnotationWithTextColleague.C

```diff
@@ -103,12 +124,103 @@ hasKeyMatch(char const *fmtStr, int idx)
     for (int ki = 0; ki < nkeys; ki++)
     {
         char const *key = keysAndFmts[ki].key;
-        if (!strncmp(&fmtStr[idx], key, strlen(key)))
+        if (!std::strncmp(&fmtStr[idx], key, strlen(key)))
             return ki;
     }
     return -1;
 }
 
+// ****************************************************************************
+// Function: GetStrFromTAFile 
+//
+// Open and read specified text annotation file (e.g. itafile1.txt,
+// ftafile2.txt or stafile3.txt) up to specified line. First try in ~/.visit,
+// then in $TMPDIR/$USER and finally in /var/tmp/$USER.
+//
+// Mark C. Miller, Thu Feb  9 11:29:30 PST 2023
+//
+// ****************************************************************************
+
+static char const*
+GetStrFromTAFile(char type, int ifile, int line)
+{
+    static char retval[256]; // whatever is returned is copied immediately
+
+    // Resolve possible directories 
+    const std::string dotVisItDir = std::string(Environment::get("HOME")) + "/.visit";
+    const std::string envTmpDir = std::string(Environment::get("TMPDIR")) + "/" +
+                                  std::string(Environment::get("USER"));
```

Comment:
```suggestion
#ifdef _WIN32
       const std::string envTmpDir = std::string(Environment::get("TMP"));
#else
        const std::string envTmpDir = std::string(Environment::get("TMPDIR")) + "/" +
                                  std::string(Environment::get("USER"));
#endif
```
The TMP (or TEMP) environment variables on Windows will return the temp dir (full path) in the current user's profile.